3IPs/3ip-0.md

```diff
@@ -0,0 +1,15 @@
+---
+3ip: 0
+title: 3IP Purpose and Guidelines
+status: Active
+author: Joel Thorstensson <oed@3box.io>
+created: 2019-01-16
+---
+
+## What is a 3IP?
+
+3IP stands for 3Box Improvement Proposal. A 3IP document contains information describing new features or other types of changes to the 3Box community. They are intended to be similar to EIPs and BIPs of Ethereum and Bitcoin respectively, but we are starting of with less formalities for now.
```

Comment:
typo // change to: "..., but 3IPs won't maintain the same level of formality as the others – for now."